### ISSUE TITLE
Suggestion: `yes to all` option for specific permission prompts like 'env'

### ISSUE THREAD

**Issue by GJZwiers** [denoland/deno#16550]
Some Deno modules need permissions to many different things, e.g. lots of different environment variables. When I'm working with a module that I don't know yet and using the permission prompts I sometimes have to confirm a lot of different prompts. When this happens usually I cancel `deno run` and retry with an option like `--allow-env`, but this is a bit of a cumbersome developer experience because I never quite know in advance how many prompts there are going to be (should I keep going with the prompts or re-run with the --allow-* flag?). For example this is how it can sometimes look on the terminal:

```
Download ⠴ https://deno.land/x/aws_api/services/sts/mod.ts
Warning Implicitly using latest version (v0.7.0) for https://deno.land/x/aws_api/services/sts/mod.ts
✅ Granted env access to "AWS_REGION".
✅ Granted env access to "AWS_ACCESS_KEY_ID".
✅ Granted env access to "AWS_SECRET_ACCESS_KEY".
✅ Granted env access to "AWS_SESSION_TOKEN".
✅ Granted env access to "AMAZON_ACCESS_KEY_ID".
✅ Granted env access to "AMAZON_SECRET_ACCESS_KEY".
✅ Granted env access to "AMAZON_SESSION_TOKEN".
✅ Granted env access to "AWS_SHARED_CREDENTIALS_FILE".
✅ Granted env access to "HOME".
✅ Granted env access to "AWS_PROFILE".
✅ Granted read access to "undefined/.aws/credentials".
✅ Granted env access to "AWS_CONTAINER_CREDENTIALS_RELATIVE_URI".
✅ Granted env access to "AWS_CONTAINER_CREDENTIALS_FULL_URI".
✅ Granted env access to "AWS_CONTAINER_SERVICE_ENDPOINT".
⚠️  ┌ Deno requests env access to "AWS_CONTAINER_AUTHORIZATION_TOKEN".
   ├ Run again with --allow-env to bypass this prompt.
   └ Allow? [y/n] (y = yes, allow; n = no, deny) >
```

It would be nice to have a 'yes to all' option during the prompts so that all prompts for the same permission like 'env' or 'read' are accepted. Something like:

```
⚠️  ┌ Deno requests env access to "AWS_REGION".
   ├ Run again with --allow-env to bypass this prompt.
   └ Allow? [y/n/A] (y = yes, allow; n = no, deny, A = yes to all) > <-- accept all 'env' prompts from this point
```

The permissions being granted should then still be shown so that users can figure out the (minimal) permissions for a production script or Docker image.

**Comment by sigmaSd** [comment]
Just a workaround for now, I have a script that says yes to all prompts https://github.com/sigmaSd/Minimize (in order to figure out the minimal perms)

 you can modify the code to make it says yes only to the permission you need

**Comment by bartlomieju** [comment]
FYI we had this feature some time ago and I was thinking of bringing it back when refreshing the permissions prompt. I'll be happy to merge a PR that implements it again.

**Comment by iuioiua** [comment]
I'll give this a crack.

**Comment by bartlomieju** [comment]
https://github.com/denoland/deno/pull/16975 is a prerequisite for this work.

**Comment by bartlomieju** [comment]
@iuioiua you should now be unblocked on this - I've added `PromptResponse` enum that can be extended with `AllowAll` and `DenyAll` variants that could affect permissions further.
